### ISSUE TITLE
Bug: Accessibility > Help button should have discernible text

### ISSUE THREAD

**Issue by ogi-m** [mattermost/focalboard#2226]
Element is in tab order and does not have accessible text ([more info](https://dequeuniversity.com/rules/axe/4.3/link-name?application=AxeChrome)).
Fix at least one (1) of these issues:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

Element location:
`.help-button`

Element source:
`<a href="https://www.focalboard.com/fwlink/doc-boards.html?v=0.12.0" target="_blank" rel="noreferrer" class="GlobalHeaderComponent__button help-button"><i class="CompassIcon icon-help-circle-outline HelpIcon"></i></a>`

![help_button](https://user-images.githubusercontent.com/15314183/152164236-c2d6627b-6156-4f00-9579-36a4bc3121fb.jpg)
